### DISCUSSION TITLE
How to access ComputedProperties from <script>?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#16120] [General - Components / Directives / etc]
The [documentation for QInput (v 2.13.3)](https://quasar.dev/vue-components/input/#qinput-api) mentions in the tab `ComputedProps` the existence of `hasError`. How can access this property from within my `<script setup lang="ts">` section? 

I guess this is a computed property (per its name) that I could `watch()` to trigger actions as soon as it changes to `true`.

**Comment by wsw70** [comment]
Found it!

This is the [Vue3 reference](https://vuejs.org/guide/typescript/composition-api.html#typing-component-template-refs). You need to add a reactive `ref="referenceOfThatComponent"` and the content is available as `referenceOfThatComponent.value.hasError`

**Reply by rstoenescu** [reply]
Also in our docs: https://quasar.dev/start/how-to-use-vue#handling-vue-methods

**Reply by maxiride** [reply]
Vue 3.5+ added `useTemplateRef()`

Docs could be slightly updated to mention it and point people to the right direction of the new method.
